frappe/website/js/web_form.js

```diff
@@ -281,14 +281,10 @@ frappe.ready(function() {
 			callback: function(data) {
 				if(!data.exc) {
 					frappe.doc_name = data.message;
-					if(!frappe.login_required) {
```

Comment:
Either ways, it should affects, commens, introduction and other sections.

Please show a screenshot / gif of what you are trying to fix.